src/plugins/github/fetchGithubRepo.js

```diff
@@ -203,18 +211,72 @@ function errorDisposition(
     return {type: "RETRY", err: e};
   }
   if (e.type === "RATE_LIMIT_EXCEEDED") {
-    // Wait in 15-minute increments. TODO(@wchargin): Ask GitHub
-    // when our token resets (`{ rateLimit { resetAt } }`) and wait
-    // until just then.
-    const delayMs = 15 * 60 * 1000;
-    return {type: "WAIT", until: new Date(Date.now() + delayMs), err: e};
+    try {
+      const nominalRefreshTime = await quotaRefreshAt(token);
+      const refreshTime = _resolveRefreshTime(new Date(), nominalRefreshTime);
+      console.warn(
+        "rate limit exceeded; waiting for refresh at " +
```

Comment:
since the messages might get interleved from a few different plugins' loads, this message could show up immediately below (say) a Discord plugin load task starting. therefore, we should specify "GitHub rate limit exceeded; waiting..."

Reply:
Sure; done.